test/utils.ts

```diff
@@ -116,6 +117,29 @@ export function getFeedPostsBatch(length = 10) {
   return posts
 }
 
+export function getLegacyFeedPostsBatch(length = 10) {
+  const posts: OBSSStorage.LegacyPostStruct[] = []
+
+  for (let i = 0; i < length; i++) {
+    posts.push({
+      post: {
+        author: `0x000000000000000000000000000000000000000${i}`,
```

Comment:
If we really need this `0x0000000000000000000000000000000000000000`, we should extract it into separate file and re-use